de-lib/src/main/java/org/iplantc/de/client/services/impl/MessageServiceFacadeImpl.java

```diff
@@ -56,27 +55,25 @@ protected Counts convertFrom(final String json) {
     }
 
     private final NotificationAutoBeanFactory notesFactory;
-    private final DEProperties deProperties;
     private final DiscEnvApiService deServiceFacade;
     private final UserInfo userInfo;
+    private final String NOTIFICATIONS = "org.iplantc.services.notifications";
```

Comment:
Ah, yes, I forgot to mention that in my PR.  After the long break, I'm really glad you caught that because I would've forgotten 😄